core/src/main/java/hivemall/utils/lang/StringUtils.java

```diff
@@ -172,12 +172,17 @@ public static void clear(@Nonnull final StringBuilder buf) {
 
     public static String concat(@Nonnull final List<String> list, @Nonnull final String sep) {
```

Comment:
@myui I guess you originally assumed this method behaves in a similar way to [what `org.apache.commons.lang3.StringUtils.join` does](https://github.com/apache/commons-lang/blob/1571050a196198f336ae487ee3b6df629d3ee9da/src/main/java/org/apache/commons/lang3/StringUtils.java#L4106-L4150). However, the original code appends a separator even at the end of result string as:

- expected: `concat(["a", "b", "c"], "-")` => `a-b-c`
- actual: `concat(["a", "b", "c"], "-")` => `a-b-c-`

So, I fixed the method in 796d388c36c520858b6e61deb34100cb9201e5fa. Is this okay? If my assumption was incorrect, I revert the modification and introduce alternative method `StringUtils.join()`.

Reply:
former is expected.

Reply:
okay, so I keep the update here

Reply:
done